libs/event-manager/web-client/data-access/src/lib/firebase/firebase.service.ts

```diff
@@ -0,0 +1,493 @@
+import { inject, Injectable } from '@angular/core';
```

Comment:
FirebaseService is a name that's too generic. It would be ideal if we could read the name of the Service and safely assume what's its purpose. For this case something like EventFirebaseService would be ideal imho.